repose-aggregator/components/filters/ip-classification/src/main/resources/META-INF/schema/config/ip-classification.xsd

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+  Repose
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Copyright (C) 2010 - 2015 Rackspace US, Inc.
+  _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:html="http://www.w3.org/1999/xhtml"
+           xmlns="http://docs.openrepose.org/repose/ip-classification/v1.0"
+           targetNamespace="http://docs.openrepose.org/repose/ip-classification/v1.0"
+           elementFormDefault="qualified"
+           attributeFormDefault="unqualified">
+
+    <xs:element name="ip-classification" type="IpClassificationConfig"/>
+
+    <xs:complexType name="IpClassificationConfig">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>The root configuration for the Repose IP Classification filter configuration file.</html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:sequence>
+            <xs:element name="user-header" type="UserHeaderType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="group-header" type="GroupHeaderType" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="group" type="GroupType" minOccurs="1" maxOccurs="unbounded"/>
+        </xs:sequence>
+    </xs:complexType>
+
+    <!-- because this became a complex type now, I cannot use defaults. Have to put them in the scala :( -->
+    <!-- well, the quality could be optional, but not the name, and if the name isn't there, the quality isn't there either -->
+    <xs:complexType name="HeaderType">
+        <!-- quality attribute follows the default from ip-identity -->
+        <xs:simpleContent>
+            <xs:extension base="xs:string">
+                <xs:annotation>
+                    <xs:documentation>
+                        <html:p>The header name to put the classification label in</html:p>
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:attribute name="quality" type="doubleBetweenZeroandOne" use="optional" default="0.4">
```

Comment:
This was updated and then had to be backed out because it didn't work. After discussing this with @jorgew, the decision was to not use either the deprecated `redefine` or the newer `override` capabilites of XML Schemas due to their complexities involved, but to instead leave the `name` attribute with a defined default in both of the extending types.